0128-energy-subnetwork.md

```diff
@@ -297,6 +309,25 @@ Oracles within the ENERGY subnetwork ensures data integrity, pricing accuracy, a
 - Gather and process market data to update pricing in given periods, reflecting changes in supply and demand, regional pricing variations, and emergency energy needs. This ensures that all transactions within the VPP are conducted at fair market rates, maximizing returns for DER hosts and minimizing costs for Clients.
 - Calculate and distribute rewards to DER hosts based on the baseline rewards.
 
+## ENERGY Subnetwork Governance
+The ENERGY subnetwork will use the current vote escrowed token model for Helium tokens and HNT stakers will be able to delegate HNT to ENERGY as they are able to delegate to IOT and MOBILE. ENERGY token holders will be able to lock up ENERGY tokens as veENERGY voting power for ENERGY subnetwork governance. There are no rewards for locking up ENERGY as veENERGY at this time.
+
+As a subnetwork in Helium, ENERGY will participate in the network utility score described in [HIP 51's "Omni Protocol PoC Incentive Model" section](https://github.com/helium/HIP/blob/main/0051-helium-dao.md#omni-protocol-poc-incentive-model) and subsequent [HIP 88](https://github.com/helium/HIP/blob/main/0088-adjustment-of-dao-utility-a-score.md).
+* ${V}$ is the staked HNT delegated to ENERGY
+* ${D}$ is the DCs burned
```

Comment:
As payment for ENERGY services, and sensor traffic

Reply:
yes, I guess the $10 dc burned when onboarding a gateway should not be counted?